tests/scripts/github-action-helper.sh

```diff
@@ -89,7 +89,7 @@ function use_local_disk_for_integration_test() {
 }
 
 function create_partitions_for_osds() {
-  tests/scripts/create-bluestore-partitions.sh --disk "$BLOCK" --osd-count 2
+  tests/scripts/create-bluestore-partitions.sh --disk "$BLOCK" --osd-count 3
```

Comment:
You need to change this too https://github.com/rook/rook/blob/24d022eebb3a3f67391c69c06aa8ae8d1e3232a3/tests/scripts/create-bluestore-partitions.sh#L54-L55

Just set to something like `4096M`.

And we should be able to have 3 OSDs easily.